flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -177,6 +177,56 @@ SqlDescribeDatabase SqlDescribeDatabase() :
 
 }
 
+SqlAlterFunction SqlAlterFunction() :
+{
+    SqlIdentifier functionName = null;
+    SqlCharStringLiteral functionClassName = null;
+    SqlCharStringLiteral functionLanguage = null;
+    SqlParserPos startPos;
+    boolean ifExists = false;
+    boolean isSystemFunction = false;
+}
+{
+    <ALTER>
+    <TEMPORARY>
```

Comment:
is <TEMPORARY> optional for alter function? My understanding is that both <TEMPORARY> and <SYSTEM> should be optional. The grammar here suggest <TEMPORARY> is required.